corehq/apps/hqpillow_retry/static/hqpillow_retry/js/single.js

```diff
@@ -0,0 +1,20 @@
+hqDefine("hqpillow_retry/js/single", function() {
+    function performAction(action) {
+        if (action === 'delete') {
+            var r=confirm("Are you sure you want to delete this error log?");
+            if (!r) {
+                return;
+            }
+        }
+        $("input[name=action]").val(action);
+        $("#actionform").submit();
+    }
+    $(function(){
```

Comment:
probably just revealing my ignorance but how does this get called? does just passing it through `hqDefine` resolve the `onready` thing at the proper time?

Reply:
yep, `hqDefine` takes care of it

Reply:
neat. :tophat: